app/controllers/actions/GroupAction.scala

```diff
@@ -124,14 +124,14 @@ class GroupAction @Inject()
 
   // TODO use withGroupSummaryForAuthorisedOptedAgent or withAccessGroupForAuthorisedOptedAgent
   def withTaxGroupForAuthorisedOptedAgent(groupId: String, isCustom: Boolean = true)
-                                         (body: TaxGroup => Future[Result])
+                                         (body: (TaxGroup, Arn) => Future[Result])
                                          (implicit ec: ExecutionContext,
                                           hc: HeaderCarrier,
                                           request: MessagesRequest[AnyContent],
                                           appConfig: AppConfig): Future[Result] = {
     authAction.isAuthorisedAgent { arn =>
       isOptedIn(arn) { _ =>
-        taxGroupService.getGroup(groupId).flatMap(_.fold(groupNotFound)(body(_)))
+        taxGroupService.getGroup(groupId).flatMap(_.fold(groupNotFound)(body(_, arn)))
```

Comment:
Isn't this now the same as withAccessGroupForAuthorisedOptedAgent(groupId, isCustom = false)

the `isCustom: Boolean = true` here doesn't make any sense 😆